src/common/cryptoutils.cpp

```diff
@@ -122,7 +122,9 @@ RetWithError<SharedPtr<crypto::PrivateKeyItf>> CertLoader::LoadPrivKeyByURL(cons
 RetWithError<UniquePtr<pkcs11::SessionContext>> CertLoader::OpenSession(
     const String& libraryPath, const String& token, const String& userPIN)
 {
-    auto library = mPKCS11->OpenLibrary(libraryPath);
+    const char* fixedPath = libraryPath.IsEmpty() ? cDefaultPKCS11Library : libraryPath.Get();
```

Comment:
Use `CStr` instead `Get`. It could be put directly to `OpenLibrary` without intermediate var. Or rename intermediate var. `fixedPath` sounds ambiguous.

Reply:
done